lxd/db/cluster/update.go

```diff
@@ -65,6 +65,49 @@ var updates = map[int]schema.Update{
 	26: updateFromV25,
 	27: updateFromV26,
 	28: updateFromV27,
+	29: updateFromV28,
+}
+
+// Add content type field to storage volumes
+func updateFromV28(tx *sql.Tx) error {
```

Comment:
@freeekanayaka can you look at the DB bits (both schema update and other changes) to see if this all looks correct given recent changes?

Reply:
Looks good to me, except from `GetStorageVolumeContentType` (see the comment), but that code area was already sub-optimal before this PR (it was one of the spots not covered by my first pass).